src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/Publisher.java

```diff
@@ -55,7 +56,11 @@ public Publisher(int evalIntervalSeconds, Collator collator) {
 
   @Override
   public EmptyFlowUnit operate() {
-    // TODO: Need to add dampening, avoidance, state persistence etc.
+    return new EmptyFlowUnit(Instant.now().toEpochMilli());
+  }
+
+  public void compute(FlowUnitOperationArgWrapper args) {
```

Comment:
It might be a good idea to add javadocs for all public methods.

Reply:
Added javadoc